code/modules/admin/verbs/playsound.dm

```diff
@@ -24,7 +24,7 @@
 	admin_sound.status = SOUND_STREAM
 	admin_sound.volume = vol
 
-	var/res = alert(usr, "Show the title of this song to the players?",, "No", "Yes", "Cancel")
+	var/res = alert(usr, "Show the title of this song to the players?",, "Yes","No", "Cancel")

```

Comment:
you should all be ashamed of this line of code tbh

Reply:
Scared?

Reply:
wrong